spec/spec_helper.rb

```diff
@@ -5,6 +5,8 @@
 
 require 'rspec'
 require 'twitter_cldr'
+require 'coveralls'
+Coveralls.wear!
```

Comment:
trying to figure this out in Ruby land myself:
https://coveralls.zendesk.com/hc/en-us/articles/201769485-Ruby-Rails

Looks like it should magically post the results after running specs